app/src/lib/dispatcher/dispatcher.ts

```diff
@@ -50,6 +50,7 @@ import { Shell } from '../shells'
 import { CloneRepositoryTab } from '../../models/clone-repository-tab'
 import { validatedRepositoryPath } from '../../lib/stores/helpers/validated-repository-path'
 import { BranchesTab } from '../../models/branches-tab'
+import { FetchType } from '../../lib/stores/app-store'
```

Comment:
:lipstick: this can be `../../lib/stores` like the other imports

Reply:
Looks like there is some more cleanup we can do here... Noticed there are other places we are not using `/lib/stores`. I'll fix that in a refactor later in life 👍